### PR TITLE
Improve verifying from fingerprint manager

### PR DESCRIPTION
**Pull request by c633**
* Part of #59 
* Offer all methods for verifying
* Update status of fingerprint after verification.

### CODE REVIEW

File: chrome/content/finger.js [hunk 1]
```diff
@@ -102,15 +102,20 @@ var otrFinger = {
...
         finger.screenname
       );
+      context = otr.getContext(uiConv.target)
     } catch (e) {}
-    let win = window.openDialog(authDialog, name, features, "pref", uiConv, finger);
+    let win = window.openDialog(authDialog, name, features, (uiConv ? "start" : "pref"), uiConv, finger);
     finger.trust = otr.isFingerprintTrusted(finger.fpointer);
+    if (context) {
+      finger.status = otr.getStatus(otr.getTrustLevel(context));
+    }
```
Comment (arlolra) [inline chrome/content/finger.js]: Let's make this test the same as above,

```
if (uiConv) {
  let context = otr.getContext(uiConv.target);
  finger.status = otr.getStatus(otr.getTrustLevel(context));
}
```

File: chrome/content/otr.js [hunk 2]
```diff
@@ -290,16 +305,7 @@ var otr = {
...
-            if (level === otr.trustState.TRUST_PRIVATE) {
-              best_level = otr.trustState.TRUST_PRIVATE;
-            } else if (level === otr.trustState.TRUST_UNVERIFIED
-                && best_level !== otr.trustState.TRUST_PRIVATE) {
-              best_level = otr.trustState.TRUST_UNVERIFIED;
-            } else if (level === otr.trustState.TRUST_FINISHED
-                && best_level === otr.trustState.TRUST_NOT_PRIVATE) {
-              best_level = otr.trustState.TRUST_FINISHED;
-            }
+            best_level = this.getTrustLevel(new Context(context_itr));
```
Comment (arlolra) [inline chrome/content/otr.js]: Can we use `otr` instead of `this`?

File: chrome/content/otr.js [hunk 3]
```diff
@@ -309,7 +315,7 @@ var otr = {
           account: wContext.account,
           protocol: wContext.protocol,
           trust: trust,
-          status: used ? getStatus(best_level) : _("trust.unused"),
+          status: used ? this.getStatus(best_level) : _("trust.unused"),
```
Comment (arlolra) [inline chrome/content/otr.js]: Same.

File: chrome/content/finger.js (hunk 1 above)
Comment (c633) [inline chrome/content/finger.js]: @arlolra fixed.

**Comment by arlolra** [comment]
Thanks!

Merged in 42cbf4ef

**Pull request closed without merging**